coriolis/tests/conductor/rpc/test_server.py

```diff
@@ -806,3 +808,471 @@ def convert_to_task(task_config):
                 self.server._check_execution_tasks_sanity(
                     execution, initial_task_info
                 )
+
+    @mock.patch.object(copy, "deepcopy")
+    @mock.patch.object(
+        server.ConductorServerEndpoint,
+        "get_replica_tasks_execution"
+    )
+    @mock.patch.object(
+        server.ConductorServerEndpoint,
+        "_begin_tasks"
+    )
+    @mock.patch.object(
+        server.ConductorServerEndpoint,
+        "_set_tasks_execution_status"
+    )
+    @mock.patch.object(
+        server.ConductorServerEndpoint,
+        "_minion_manager_client"
+    )
+    @mock.patch.object(db_api, "add_replica_tasks_execution")
+    @mock.patch.object(db_api, "update_transfer_action_info_for_instance")
+    @mock.patch.object(
+        server.ConductorServerEndpoint,
+        "_check_execution_tasks_sanity"
+    )
+    @mock.patch.object(
+        server.ConductorServerEndpoint,
+        "_create_task"
+    )
+    @mock.patch.object(uuid, "uuid4")
+    @mock.patch.object(models, "TasksExecution")
+    @mock.patch.object(
+        server.ConductorServerEndpoint,
+        "_check_minion_pools_for_action"
+    )
+    @mock.patch.object(
+        server.ConductorServerEndpoint,
+        "_check_replica_running_executions"
+    )
+    @mock.patch.object(
+        server.ConductorServerEndpoint,
+        "_check_reservation_for_transfer"
+    )
+    @mock.patch.object(
+        server.ConductorServerEndpoint,
+        "_get_replica"
+    )
+    @ddt.file_data("data/execute_replica_tasks_config.yml")
+    @ddt.unpack
+    def test_execute_replica_tasks(
+            self,
+            mock_get_replica,
+            mock_check_reservation,
+            mock_check_replica_running_executions,
+            mock_check_minion_pools_for_action,
+            mock_tasks_execution,
+            mock_uuid4,  # pylint: disable=unused-argument
+            mock_create_task,
+            mock_check_execution_tasks_sanity,
+            mock_update_transfer_action_info_for_instance,
+            mock_add_replica_tasks_execution,
+            mock_minion_manager_client,
+            mock_set_tasks_execution_status,
+            mock_begin_tasks,
+            mock_get_replica_tasks_execution,
+            mock_deepcopy,
+            config,
+            expected_tasks,
+    ):
+        has_origin_minion_pool = config.get("origin_minion_pool", False)
+        has_target_minion_pool = config.get("target_minion_pool", False)
+        shutdown_instances = config.get("shutdown_instances", None)
```

Comment:
`False` should have been used, instead of `None`. I think I did it just out of habit of working with some other function parameters in the unit tests like `depends_on` which can be `None`. All instances of `shutdown_instances` have been set to `False` in the unit tests.